src/chunks/founders-template-chunk/founders-all-template-chunk/FoundersAllTemplateChunk.tsx

```diff
@@ -0,0 +1,204 @@
+              connects employers with highly qualified and diverse candidates,
+              helping them find the best talent to grow their team within budget
+              and business need. We democratize access to internships and
+              entry-level jobs, empowering students and graduate to build skills
+              and experience to become part of the future talent pipeline in the
+              technology industry.
+            </p>
+          </div>
+
+          <div className="flex md:items-center md:mt-[7rem] lg:mt-[6rem]">
+            {/* divider */}
+            <div className="border-[2px] border-light-blue lg:h-[200px] md:h-[230px] h-full mx-[0.6rem]  md:mx-[1.5rem]"></div>
+
+            <h1 className="text-[72px]  text-[#26A4FF] font-[600] leading-[79px]">
+              2
+            </h1>
+          </div>
+        </div>
+
+        {/* vission section */}
+        <div className="flex  md:items-center  justify-end md:w-[704px] lg:h-[200px] md:mt-[5rem] mt-[3rem] lg:w-[824px]  gap-[10px] md:mb-0 mmb-[4rem] lg:ml-[10rem]">
+          <div>
+            <h1 className="text-[72px] text-[#002360] font-[600] leading-[79px]">
+              3
+            </h1>
+          </div>
+
+          {/* divider */}
+          <div className="border-[2px] border-light-blue md:h-[200px] md:mx-[1.5rem]"></div>
+
+          <div className=" md:h-[198px] gap-[30px]">
+            <div className="flex  items-center gap-[20px]">
+              <div className="w-[30px] h-[30px] rounded-[50%] justify-center items-center flex bg-[#D6DDEB] cursor-pointer">
+                <Image
+                  src={vision_gilf}
+                  alt=""
+                  className="w-[25px] h-[25px] "
+                />
+              </div>
+
+              <ul className="flex gap-[10px]">
+                <li className="text-grey font-clash font-[600] text-[24px] leading-[42px]">
+                  Our
+                </li>
+                <li className="text-light-blue font-clash font-[600] text-[24px] leading-[42px]">
+                  Story
+                </li>
+              </ul>
+            </div>
+
+            <p className="font-epilogue text-[18px] leading-[32px] text-grey font-[400] gap-[10px] pt-[1.4rem]">
+              We envision a world where African talent is recognized and valued
+              globally, where skilled professionals can connect with
+              opportunities that match their potential, and where businesses can
+              seamlessly access the diverse talent they need to thrive.
+            </p>
+          </div>
+        </div>
+      </div>
+
+      {/*What we offer */}
+      <div className="flex flex-col items-center lg:my-[5rem] mt-[4rem] md:mt-auto md:my-[3rem]">
+        {/* Heading */}
+        <div className="gap-[10px] py-[3rem]">
+          <h1 className="font-clash font-[600] text-[32px] leading-[38px] text-[#1B1B1B]">
```

Comment:
move the what we offer and founder section to another page, this file has a lot of lines already